comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go

```diff
@@ -465,6 +465,10 @@ func (c *WorkloadMetaCollector) handleECSTask(ev workloadmeta.Event) []*types.Ta
 		addResourceTags(taskTags, task.Tags)
 	}
 
+	if task.ServiceName != "" {
+		taskTags.AddLow(tags.EcsServiceName, strings.ToLower(task.ServiceName))
```

Comment:
Could be good to add a test for this?

Reply:
Please add a test for it as @robertjli suggested 🙏 

I recently added some tests for tags that were not being tested. Maybe these PRs can be useful as an example: https://github.com/DataDog/datadog-agent/pull/26823 https://github.com/DataDog/datadog-agent/pull/26822

Reply:
thanks for the example ! https://github.com/DataDog/datadog-agent/pull/28660/commits/ee795885441008e2e1210b872d34dafcb749dbf4